### PR TITLE
JAMES-2587 Adding debian/rpm packaging for Cassandra-RabbitMQ

### PR DESCRIPTION
**Pull request by trantienduchn**
> Also, we should be adding debian/rpm packaging - maybe in another pr

from #1892

### CODE REVIEW

File: dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/quota.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
```
Comment (chibenwa) [inline dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/quota.xml]: This file is not needed. Please remove it.

File: dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/rabbitmq.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+uri=amqp://rabbitmq:5672
```
Comment (chibenwa) [inline dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/rabbitmq.properties]: Use `127.0.0.1` for rabbit host too

File: dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/quota.xml (hunk 1 above)
Comment (trantienduchn) [reply dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/quota.xml]: There is a `quota.xml` in `packaging/guice/cassandra`. I guess I should remove it also

**Comment by chibenwa** [comment]
You need to add explainations in `Generating a package for James` section of the README

**Comment by trantienduchn** [comment]
```
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR] Errors: 
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR]   MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [INFO] 
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR] Tests run: 179, Failures: 0, Errors: 1, Skipped: 0
```
test this please

**Comment by trantienduchn** [comment]
```
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR] Errors: 
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR]   JpaSelectTest.testSelectUnseenUS » Runtime java.io.IOException: Failed to list...
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [INFO] 
[8e4592859a72ff55b36db6ebc3293aea3d64c01e] [ERROR] Tests run: 315, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by trantienduchn** [comment]
test this please

**Comment by trantienduchn** [comment]
Close this. Regards to new packaging steps from #1897

**Pull request closed without merging**